scipy/sparse/linalg/_isolve/tests/test_iterative.py

```diff
@@ -404,21 +411,22 @@ def test_atol(solver):
             x, info = solver(A, b, M1=M, M2=M2, tol=tol, atol=atol)
         else:
             x, info = solver(A, b, M=M, tol=tol, atol=atol)
-        assert_equal(info, 0)
 
-        residual = A.dot(x) - b
+        assert info == 0
```

Comment:
I see that the replacement of scalar `assert_equal` was only done here; I have a follow-up commit ready (if desired) that removes all usage of that function from this module